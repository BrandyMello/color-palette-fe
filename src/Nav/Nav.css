.navBar {}

.logo {
  height: 200px;
  width: auto;
  position: absolute;
  text-align: center;
  top: 17.5%;
  right: 19%;
}

h1 {
  font-size: 4em;
  text-align: center;
}

input {
  height: 30px;
  width: 120px;
  border-radius: 3% 0 0 3%;
}

button {
  height: 34px;
}

.homeButton {
  margin: 50px;
}