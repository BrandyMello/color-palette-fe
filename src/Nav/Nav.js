import React, { Component } from 'react'
import './Nav.css'

class Nav extends Component {
  constructor() {
    super()
  }
  render() {
    return (
      <div>
        <h1>Color Picker</h1>
        <p>insert logo</p>
        <form>
          <input placeholder= "search"/>
          <button type="submit">Submit</button>
        </form>
      </div>
    )
  }
}

export default Nav