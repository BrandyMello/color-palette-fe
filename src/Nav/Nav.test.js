import React from "react";
import { shallow } from "enzyme";
import Nav from "./Nav";

describe("Nav", () => {
  it("renders the title of the app in <h1> tags", () => {
    const wrapper = shallow(<Nav title="Bananas" />);
    const title = <h1>Color Palette</h1>;

    expect(wrapper.contains(title)).toEqual(true);
  });
  it("renders the image of the logo for the app in an <img> tag", () => {
    const wrapper = shallow(
      <Nav
        img="https://user-images.githubusercontent.com/46384968/66802881-2ce58a80-eedb-11e9-97f2-e60bf5a8e28b.png"
        className="logo"
      />
    );
    const img = (
      <img
        src="https://user-images.githubusercontent.com/46384968/66802881-2ce58a80-eedb-11e9-97f2-e60bf5a8e28b.png"
        className="logo" alt="Frida"
      ></img>
    );

    expect(wrapper.contains(img)).toEqual(true);
  });
  it("should match the snapshot", () => {
    const wrapper = shallow(
      <Nav
        title="Color Palette"
        img="https://user-images.githubusercontent.com/46384968/66802881-2ce58a80-eedb-11e9-97f2-e60bf5a8e28b.png"
        className="logo"
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
