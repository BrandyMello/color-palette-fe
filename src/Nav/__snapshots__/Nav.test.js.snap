// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
<div
  className="navBar"
>
  <h1>
    Color Palette
  </h1>
  <img
    className="logo"
    src="https://user-images.githubusercontent.com/46384968/66802881-2ce58a80-eedb-11e9-97f2-e60bf5a8e28b.png"
  />
  <form />
</div>
`;
